sidecar.sh

```diff
@@ -23,40 +23,36 @@ populate_notebook() {
         filesize=$(echo $j | jq -r .file_size)
 
         # Need to add a literal newline character that's why the quote is ending on next line
-        drs_pull="!gen3 drs-pull object $obj
-
-"
+        drs_pull="!gen3 drs-pull object $obj"
         # Need to add a literal newline character that's why the quote is ending on next line
-        jq --arg cmd "# File name: $filename - File size: $filesize
-" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
+        jq --arg cmd "# File name: $filename - File size: $filesize" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
         mv "$FOLDER/data.tmp" "$FOLDER/data.ipynb"
 
         jq --arg cmd "$drs_pull" '.cells[5].source += [$cmd]' "$FOLDER/data.ipynb" > "$FOLDER/data.tmp"
         mv "$FOLDER/data.tmp" "$FOLDER/data.ipynb"
 
-
     done
     log "Done populating notebook"
 }
 
 function populate() {
     log "querying manifest service at $GEN3_ENDPOINT/manifests"
-    MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests")
+    MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
```

Comment:
Shouldn't it be https://$GEN3_ENDPOINT/manifests ?

Reply:
it will be redirected to `https://$GEN3_ENDPOINT/manifests/` even if you try to hit `https://$GEN3_ENDPOINT/manifests`